<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UDPClient</class>
 <widget class="QDialog" name="UDPClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>UDPClient Connect First</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/circuitry.png);

</string>
  </property>
  <widget class="QPushButton" name="ON">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#ON {
background-color: transparent;
border-image: url(:images/OFF.png);
background: none;
border: none;
background-repeat: none;
}
#ON:pressed
{
border-image: url(:images/ON.png);
}
#ON:onClicked
{
border-image: url(:images/ON.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="LED">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>124</width>
     <height>305</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#LED {
background-color: transparent;
border-image: url(:images/LEDOFF.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="TOGGLE">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox#TOGGLE {font-size:14pt;font-weight:bold;color: rgb(93,94,95)}
#QCheckBox#TOGGLE::indicator{ background-color:#FFF; }
#QCheckBox#TOGGLE{border: 2px solid #5A5A5A;background: none;}

</string>
   </property>
   <property name="text">
    <string>Toggle</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="EXIT">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#EXIT {
background-color: transparent;
border-image: url(:images/EXIT.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>108</width>
     <height>196</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="client">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);
font: 75 14pt &quot;Arial&quot;;
text-decoration: underline;
</string>
      </property>
      <property name="text">
       <string>Client:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QLabel" name="port_to">
        <property name="styleSheet">
         <string notr="true">
QLabel#port_to {font-size:11pt;font-weight:bold;color:#550000;}
</string>
        </property>
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="message_to">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#message_to {font-size:11pt;font-weight:bold;color:#550000;}</string>
        </property>
        <property name="text">
         <string>Message to:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="data_to">
        <property name="styleSheet">
         <string notr="true">QLabel#data_to {font-size:11pt;font-weight:bold;color:#550000;}</string>
        </property>
        <property name="text">
         <string>Data:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="Server">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);
font: 75 14pt &quot;Arial&quot;;
text-decoration: underline;
</string>
      </property>
      <property name="text">
       <string>Server:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="message_from">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#message_from {font-size:11pt;font-weight:bold;color:#550000;}</string>
        </property>
        <property name="text">
         <string>Message from:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="port_from">
        <property name="styleSheet">
         <string notr="true">QLabel#port_from {font-size:11pt;font-weight:bold;color:#550000;}</string>
        </property>
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="data_from">
        <property name="styleSheet">
         <string notr="true">QLabel#data_from {font-size:11pt;font-weight:bold;color:#550000;}</string>
        </property>
        <property name="text">
         <string>Data:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <widget class="QLabel" name="display_data_from">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="display_message_from">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="display_port_from">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="display_message_to">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="display_port_to">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#display_port_to {
background: white;
font-weight:bold;
border: none;
background-repeat: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="display_data_to">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#display_data_to {
background: white;
font-weight:bold;
border: none;
background-repeat: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="changeIP">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton#changeIP {
 background-color: rgb( 152, 213, 216 );
 }
 QPushButton#changeIP:pressed {
     background-color: rgb(153, 60, 7);     
 }
 QPushButton#changeIP:hover {
     background-color: rgb( 7, 146, 153);
 }</string>
   </property>
   <property name="text">
    <string>Change IP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changePort">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#changePort {
 background-color: rgb( 152, 213, 216 );
 }
 QPushButton#changePort:pressed {
     background-color: rgb(153, 60, 7);     
 }
 QPushButton#changePort:hover {
     background-color: rgb( 7, 146, 153);
 }</string>
   </property>
   <property name="text">
    <string>Change Port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IP">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Port">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>ON</zorder>
  <zorder>LED</zorder>
  <zorder>TOGGLE</zorder>
  <zorder>EXIT</zorder>
  <zorder>changeIP</zorder>
  <zorder>changePort</zorder>
  <zorder>IP</zorder>
  <zorder>Port</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
